src/atomate2/vasp/schemas/ferroelectric.py

```diff
@@ -0,0 +1,104 @@
+"""Schemas for Ferroelectric wflow."""
+
+from typing import List
+
+from pydantic import BaseModel, Field
+from pymatgen.core import Structure
+
+__all__ = ["PolarizationDocument"]
+
+
+class PolarizationDocument(BaseModel):
```

Comment:
The schema looks good. It would be nice to add some metadata about the calculations that were used to generate it. E.g., the elastic workflow has a `fitting_data` field that stores the inputs, the job uuids and jobs dirs: https://github.com/materialsproject/atomate2/blob/98f3ed50d587fc1d07848487f21df83087b9d256/src/atomate2/common/schemas/elastic.py#L82

Things to include in general are: 
1. The uuids and job dirs of the initial, final and interpolated LCALCPOL calculations. This should be in the same order as the `structures` dict.

Reply:
done